### DISCUSSION TITLE
import dpdata.deepmd error

### DISCUSSION THREAD

**Discussion by Lewis-YL** [deepmodeling/deepmd-kit#1763] [Q&A] [answered]
Hello,

I had no such problem before with deepmd v2.1.0.
After I reinstalled deepmd v2.1.1.
I have problem import dpdata.deepmd
Here is the error message:

  File "/home/ptxlab/.local/lib/python3.9/site-packages/dpdata/plugins/deepmd.py", line 1, in <module>
    import dpdata.deepmd.raw
ModuleNotFoundError: No module named 'dpdata.deepmd'

How can I fix the problem?

**Comment by njzjz** [comment, marked as answer]
... dpdata is not a dependency of deepmd-kit.

Are you sure your issue is related to the DeePMD-kit?

**Reply by Lewis-YL** [reply]
Nevermind, the problem is solved by reinstall a new version of dpdata.

Thank you.
